src/initialization/initializeTypeScript/initializeTests.ts

```diff
@@ -15,6 +17,7 @@ const initializeBuiltInTests = async () => {
         "src/**/__tests__/*.test.{ts,tsx}",
         "src/**/*.test.{ts,tsx}",
         "test/**/*.{ts,tsx}",
+        "(none)",
```

Comment:
My suggestion was to make the `const none` variable so it can be re-used. That way we don't have one location using `"none"` and one using `"(none)"`. 🙂